### PR TITLE
enh: change routing to www site for about

### PR DESCRIPTION
**Pull request by satra**
this is a precursor to changing the redirector to point directly at the new portal.

### CODE REVIEW

**Comment by satra** [comment]
@brianhelba - `dandiUrl` is being used to create the permalink.

File: web/src/components/AppBar/AppBar.vue [hunk 1]
```diff
@@ -88,7 +88,7 @@
 
 <script>
 import { loggedIn } from '@/rest';
-import { dandiUrl } from '@/utils';
+import { dandiUrl, dandiAboutUrl } from '@/utils';
```
Comment (brianhelba) [inline web/src/components/AppBar/AppBar.vue:91]:
```suggestion
import { dandiAboutUrl } from '@/utils';
```

File: web/src/components/AppBar/AppBar.vue [hunk 2]
```diff
@@ -99,6 +99,7 @@ export default {
   data() {
     return {
       dandiUrl,
```
Comment (brianhelba) [inline web/src/components/AppBar/AppBar.vue:101]:
```suggestion
```

File: web/src/components/DandiFooter.vue [hunk 3]
```diff
@@ -1,11 +1,12 @@
 <script>
-import { copyToClipboard, dandiUrl } from '@/utils';
+import { copyToClipboard, dandiUrl, dandiAboutUrl } from '@/utils';
 
 export default {
   name: 'DandiFooter',
   data: () => ({
     copied: false,
     dandiUrl,
```
Comment (brianhelba) [inline web/src/components/DandiFooter.vue:8]:
```suggestion
```

File: web/src/components/DandiFooter.vue [hunk 4]
```diff
@@ -1,11 +1,12 @@
 <script>
-import { copyToClipboard, dandiUrl } from '@/utils';
+import { copyToClipboard, dandiUrl, dandiAboutUrl } from '@/utils';
```
Comment (brianhelba) [inline web/src/components/DandiFooter.vue:2]:
```suggestion
import { copyToClipboard, dandiAboutUrl } from '@/utils';
```

**Pull request merged**